### DISCUSSION TITLE
Do you use python2?

### DISCUSSION THREAD

**Discussion by alevikpes** [UtrechtUniversity/yoda#473] [Q&A] [answered]
Hello!

I was looking at your project and noticed that you implement workarounds for installing pip2 for centos7. Do you use python2 in your project or is it something you depend on uses python2?

thank you

**Comment by stsnel** [comment, marked as answer]
Hello,

Yes, although most code is for Python 3.x, the current version of Yoda still uses Python 2 for the iRODS ruleset. This is because the version of iRODS we're using only supports Python 2, and upgrading to a new iRODS version requires major changes in the application which haven't been completed yet. We aim to remove the last Python 2 code in the next major version of Yoda.

Best regards,
Sietse
